routes/api.php

```diff
@@ -17,3 +17,5 @@
 Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+Route::apiResource('v1/products', \App\Http\Controllers\ProductController::class);
```

Comment:
Put v1 in route's group, so you don't need to repeat it in every route, make V1 Subdirectory inside controllers to separate every version controllers.

Reply:
I will move it entirely to the `RouteServiceProvider`, so we can use a different files for each version (e.g. api_v2.php) in the future.